packages/client/components/TeamPrompt/TeamPromptEditablePromptModal.tsx

```diff
@@ -0,0 +1,201 @@
+  display: 'flex',
+  justifyContent: 'flex-end'
+})
+
+const CloseIcon = styled(Icon)({
+  color: PALETTE.SLATE_600,
+  cursor: 'pointer',
+  fontSize: ICON_SIZE.MD24,
+  '&:hover': {
+    opacity: 0.5
+  }
+})
+
+const StyledRaisedButton = styled(RaisedButton)({
+  marginLeft: 16,
+  paddingTop: 12,
+  paddingBottom: 12,
+  fontSize: 18
+})
+
+const StyledCloseButton = styled(PlainButton)({
+  height: 24,
+  marginLeft: 'auto'
+})
+
+const ErrorWrapper = styled('div')({
+  color: PALETTE.ROSE_500,
+  display: 'flex',
+  alignItems: 'center',
+  fontSize: 13,
+  marginTop: 16
+})
+
+const ErrorIcon = styled(Icon)({
+  color: PALETTE.SLATE_500,
+  fontSize: ICON_SIZE.MD18,
+  marginRight: 8
+})
+
+const SUGGESTED_PROMPTS = [
+  'What’s changed with your tasks?',
+  'What will you do today?',
+  'What have you done since yesterday?',
+  'What did you learn today?',
+  'What’s got your attention today?'
+]
+
+interface Props {
+  initialPrompt: string
+  onCloseModal: () => void
+  onSubmitUpdatePrompt: (newPrompt: string) => void
+  onCompleted: () => void
+  error?: string
+}
+
+const TeamPromptEditablePromptModal = (props: Props) => {
+  const {initialPrompt, onCloseModal, onSubmitUpdatePrompt, error} = props
+  const {
+    validateField,
+    setDirtyField,
+    setValue,
+    onChange: onChangePrompt,
+    fields
+  } = useForm({
+    meetingPrompt: {
+      getDefault: () => initialPrompt,
+      validate: (rawMeetingPrompt) => {
+        return new Legitity(rawMeetingPrompt)
+          .trim()
+          .required('A prompt is required to start the activity')
+          .min(2, 'Prompts should be at least two characters long')
+      }
+    }
+  })
+
+  const displayError = fields.meetingPrompt.error ?? error
+
+  const isDesktop = useBreakpoint(Breakpoint.SIDEBAR_LEFT)
+
+  const handleSubmitUpdate = () => {
+    setDirtyField()
+    const {meetingPrompt: meetingPromptRes} = validateField()
+    if (meetingPromptRes.error) return
+    const {value: meetingPrompt} = meetingPromptRes
+    onSubmitUpdatePrompt(meetingPrompt)
+    onCloseModal()
+  }
+
+  return (
+    <StyledDialogContainer>
+      <StyledDialogTitle>
+        {'Prompt'}
+        <StyledCloseButton onClick={onCloseModal}>
+          <CloseIcon>close</CloseIcon>
+        </StyledCloseButton>
+      </StyledDialogTitle>
+      <DialogContent>
+        <TextArea
+          {...fields.meetingPrompt}
+          name={'meetingPrompt'}
+          isDesktop={isDesktop}
+          autoFocus={true}
+          maxLength={500}
+          maxRows={3}
+          onChange={onChangePrompt}
+          placeholder={'What are you working on today? Stuck on anything?'}
+        />
+        {displayError && (
+          <ErrorWrapper>
+            <ErrorIcon>info</ErrorIcon>
+            {displayError}
+          </ErrorWrapper>
+        )}
+        <SuggestedPromptsHeader>Suggestions</SuggestedPromptsHeader>
+        {SUGGESTED_PROMPTS.map((prompt, i) => (
```

Comment:
+1 optionally filter out the currently selected value if it's easy & doesn't update on keystroke